core_impl/src/main/java/io/opencensus/stats/MeasureToViewMap.java

```diff
@@ -84,47 +84,52 @@ synchronized void registerView(View view, Clock clock) {
   }
 
   // Records stats with a set of tags.
-  synchronized void record(StatsContextImpl tags, MeasureMap stats) {
+  synchronized void record(StatsContextImpl tags, MeasureMap stats, Clock clock) {
```

Comment:
There's got to be a better way then passing a clock around everywhere. Maybe we need something like Clock in StatsComponent.